lib/api/controllers/securityController.js

```diff
@@ -1024,6 +1025,19 @@ class SecurityController extends BaseController {
 
     return Bluebird.resolve(strategyFields);
   }
+
+  revokeTokens(request) {
+    assertHasId(request);
+
+    const userId = request.input.resource._id;
+    return this.kuzzle.repositories.user.load(userId)
+      .then(user => {
+        if (!user) {
+          return Bluebird.reject(new NotFoundError(`User with id ${userId} not found`));
```

Comment:
There is no need to use `Bluebird.reject` since you are already in a promise chain. If you throw an exception inside the promise chain, the entire promise will be rejected.
And also you could use the errorManger here no?

Reply:
I prefere to wait https://github.com/kuzzleio/kuzzle/pull/1333 is merged before using the errorsManager here to avoid conflicts